### DISCUSSION TITLE
Can we merge two signature matrices: one from scRNA with the LM22?

### DISCUSSION THREAD

**Discussion by bharati-mehani** [omnideconv/immunedeconv#66] [Q&A]
Hello @grst , @mlist, @federicomarini, @FFinotello, @traviscibot and all other cibersortx users,

I have a query related to CIBERSORTx, where I have a tumor specific signature matrix generated from scRNA seq data. I want to merge it with the immune signatures (LM22 matrix), in order to explore tumor and non-tumor contents together.
I am wondering if it is possible to merge two signature matrices generated by two different platforms?
Any help or guidance to solve this issue will be highly appreciated.
Thanks

**Comment by grst** [comment]
I think that question would be better directed at the cibersort authors. 

My guess would be that this doesn't work well. I would rather try to find single cell dataset that also contains immune cells and use that to generate the signature matrix with cibersortx.
